.meta-updater/src/index.ts

```diff
@@ -20,8 +21,11 @@ export default async (workspaceDir: string) => {
   if (lockfile == null) {
     throw new Error('no lockfile found')
   }
-  return {
-    'package.json': (manifest: ProjectManifest & { keywords?: string[] }, dir: string) => {
+  return createUpdateOptions({
+    'package.json': (manifest: ProjectManifest & { keywords?: string[] } | null, { dir }) => {
```

Comment:
Extension .json currently resolves to `object | null`. Might try to check if it is possible to have special case for `package.json` where it will be `ProjectManifest` (w/o `| null`).

`dir` is now part of `FormatPluginFnOptions`